### PR TITLE
[KP-766] install handle both paths and repos

### PR DESCRIPTION
**Pull request by graddaniel**
https://kaholo.atlassian.net/browse/KP-766

### CODE REVIEW

**Review by kenskyfish — DISMISSED** [review]
test and working well, including overrides

File: helm-cli.js [hunk 1]
```diff
@@ -274,11 +300,14 @@ function splitDirectory(directory) {
   ];
 }
 
-// Helm Docker image accepts commands WITHOUT the CLI name
-function sanitizeCommand(command) {
-  return command.startsWith(HELM_CLI_NAME)
-    ? command.slice(HELM_CLI_NAME.length).trim()
-    : command;
+function extractChartDirectoryFromCommand(command) {
+  const directory = command.match(ABSOLUTE_PATH_REGEXP);
```
Comment (xkcm) [inline helm-cli.js]: There exists a 'extractPathsFromCommand` function in the kaholo-plugin-library for the path extraction, it covers a broad range of possible path arguments in a command.

File: helm-cli.js [hunk 2]
```diff
@@ -233,18 +254,23 @@ ${parametersWithEnvironmentalVariablesArray.join(" ")}`;
...
+  };
 }
 
-function wrapChartVolumeDefinition(volumeDefinition, chartDirectory) {
-  const [, chartDirectoryName] = splitDirectory(chartDirectory);
+function addExtraPathToVolumeDefinition(volumeDefinition, workingDirectory) {
+  const [, workingDirectoryName] = splitDirectory(workingDirectory);
 
   /* eslint-disable no-param-reassign,no-unused-expressions */
   volumeDefinition.mountPoint.value.endsWith("/")
```
Comment (xkcm) [inline helm-cli.js:271]: Wouldn't this be cleaner to just use `if / else`? Or if you want to join two path segments, you can use `join()` function from `path` node module.

**Review by xkcm — CHANGES_REQUESTED** [review]

File: helm-cli.js (hunk 2 above)
Comment (graddaniel) [reply helm-cli.js:271]: I don't understand.

My point was to modify the existing `mountPoint.value` by appending it with the working directory's name. I've replaced the old call with direct `path.basename` call.

File: helm-cli.js (hunk 1 above)
Comment (graddaniel) [reply helm-cli.js]: Thanks, I forgot about that one.

File: helm-cli.js (hunk 2 above)
Comment (xkcm) [reply helm-cli.js:271]: I think the lines highlighting may be misleading, I've meant to address lines 272-274.

**Review by xkcm — DISMISSED** [review]

**Pull request merged**